test/built-ins/Atomics/wait/poisoned-object-for-timeout-throws.js

```diff
@@ -34,28 +31,31 @@ function getReport() {
 $262.agent.start(
   `
 $262.agent.receiveBroadcast(function (sab) {
+  
+  var int32Array = new Int32Array(sab);
+
   var poisoned = {
     valueOf: false,
     toString: false
   };
-
-  var err;
   
   try {
     Atomics.wait(int32Array, 0, 0, poisoned);
   } catch(e) {
-    err = e.constructor;
+    $262.agent.report(e.name);
```

Comment:
that's a rollback. Keep err outside, don't report anything from inside the catch block. So you guarantee you always have a .report call